### PR TITLE
Admin API docs for C chain

### PR DESCRIPTION
**Pull request by obbap1**

### CODE REVIEW

**Comment by obbap1** [comment]
@aaronbuchwald cant we just add all the performance stuff to the Admin API? the way we have it on avalanchego

File: build/avalanchego-apis/admin-api-c-chain.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# Admin API (C-Chain)
```
Comment (danlaine) [inline build/avalanchego-apis/admin-api-c-chain.md:1]: The C-Chain admin API should go on the existing C-Chain API page

File: build/avalanchego-apis/admin-api-c-chain.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+Sets the log level of the C-Chain.
+
+#### **Signature**
+
+```text
+admin.setLogLevel({level:string}) -> {success:bool}
+```
+
+* `level` is the log level to be set.
```
Comment (danlaine) [inline build/avalanchego-apis/admin-api-c-chain.md:29]: What are the log levels they can provide?

File: build/avalanchego-apis/admin-api-c-chain.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
+# Admin API (C-Chain)
+
+This API can be used for debugging. Note that the Admin API is disabled by default. To run a node with the Admin API enabled, use [command line argument](../references/command-line-interface.md#c-chain-config) `--coreth-admin-api-enabled=true`.
```
Comment (danlaine) [inline build/avalanchego-apis/admin-api-c-chain.md:3]:
```suggestion
This API can be used for debugging. Note that the Admin API is disabled by default. To run a node with the Admin API enabled, use [C-Chain config option](../references/command-line-interface.md#c-chain-config) `"coreth-admin-api-enabled":true`.
```

File: build/avalanchego-apis/admin-api-c-chain.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    "id"     :1,
+    "result" :{
+        "success":true
+    }
+}
+```
+
+### admin.startCPUProfiler
+
+Starts a CPU profile writing to the specified file
```
Comment (danlaine) [inline build/avalanchego-apis/admin-api-c-chain.md:58]: Which file does it write to? In the signature below the user doesn't give a file name

File: build/avalanchego-apis/admin-api-c-chain.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+    "id"     :1,
+    "result" :{
+        "success":true
+    }
+}
+```
+
+### admin.memoryProfile
+
+Runs a memory profile writing to the specified file.
```
Comment (danlaine) [inline build/avalanchego-apis/admin-api-c-chain.md:124]: Same about user not specifying a file

File: build/avalanchego-apis/admin-api-c-chain.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+    "id"     :1,
+    "result" :{
+        "success":true
+    }
+}
+```
+
+### admin.lockProfile
+
+Runs a mutex profile writing to the specified file.
```
Comment (danlaine) [inline build/avalanchego-apis/admin-api-c-chain.md:157]: Same about user not specifying a file

File: build/avalanchego-apis/admin-api-p-chain.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# Admin API
+# Admin API (P-Chain)
```
Comment (danlaine) [inline build/avalanchego-apis/admin-api-p-chain.md:1]: This can just be `Admin API` -- it's not P-Chain specific

**Review by danlaine — CHANGES_REQUESTED** [review]

File: build/avalanchego-apis/admin-api-c-chain.md (hunk 2 above)
Comment (obbap1) [reply build/avalanchego-apis/admin-api-c-chain.md:29]: its already listed here https://docs.avax.network/build/references/command-line-interface#c-chain-config

Screenshot 2021-11-01 at 18 58 07

should I list them again ?

File: build/avalanchego-apis/admin-api-c-chain.md (hunk 1 above)
Comment (obbap1) [reply build/avalanchego-apis/admin-api-c-chain.md:1]: Okay 👍🏾

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 8]
```diff
@@ -742,3 +742,190 @@ curl -X POST --data '{
...
+    "id"     :1,
+    "result" :{
+        "success":true
+    }
+}
+```
+
+### admin.lockProfile
+
+Runs a mutex profile writing to the `coreth_performance_c` directory.
```
Comment (obbap1) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:901]: @danlaine so it writes to this directory https://github.com/ava-labs/coreth-internal/blob/master/plugin/evm/vm.go#L691. I'm not sure how to word it perfectly

**Review by danlaine — APPROVED** [review]

**Pull request merged**